.github/workflows/ci.yml

```
@@ -282,7 +282,10 @@ jobs:
         run: |
           ccache -p || true
           cd seqan3-build
-          make -k -j2
+          if [[ "${{ matrix.build }}" =~ ^(documentation)$ ]]; then
+            make download-cppreference-doxygen-web-tag; else
+            make -k -j2
+          fi
```

```suggestion
          if [[ "${{ matrix.build }}" =~ ^(documentation)$ ]]; then
            make download-cppreference-doxygen-web-tag;
          else
            make -k -j2
          fi
```

I don't know bash enough, but is it possible to move the else to the next line?

---

Should work, we can do else on a new line.

I'll make it a backlog item

---

https://github.com/seqan/product_backlog/issues/230

---

Further above uses 

```
        run: |
          if [ "$RUNNER_OS" == "Linux" ]; then
            sudo apt-get install --yes ${{ matrix.cxx }}
          else
            brew install --force-bottle $(echo "${{ matrix.cxx }}" | sed "s/++-/cc@/g")
          fi
```

not sure why we can't use it here if we introduce this.